### ISSUE TITLE
Sprint - Dec 23 - Jan 3

### ISSUE THREAD

**Issue by dmarticus** [PostHog/posthog#26946]
# Global Sprint Planning

## 3 things that might take us down

1.
2.
3.

## Urgent incident follow-ups

https://github.com/orgs/PostHog/projects/103/views/2

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
# Team ___

**Support hero:** ___

## Retro

<!-- Grab the high and low priority items from last time and add whether that item was completed or not -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## OKR

1. OKR, status (red/yellow/green) and action points if yellow/red


### High priority

-

### Low priority / side quests

-

```

**Comment by jurajmajerik** [comment]
# Team Experiments 🧪 

## Retro


## Sprint goals
Support hero: @jurajmajerik

**Comment by daibhin** [comment]
# Team Flags

## Support

- 12-16 to 12-23 @dmarticus 
- Let's talk about how to do support for the holidays

## Retro

### Things we got done

- [x] testing/productionizing new `/flags` service @dmarticus 
    - [x] fixing P0 bugs along `/flags` hotpath 
    - [x] functional testing via django-compat integration tests
- [x] Figure out how to cut off the flags bit of `/decide` (and, in future `/flags`) when folks go over – right now we don't stop serving flags when overage happens @dmarticus 
    - going to do this in the new `/flags` endpoint

### Things we didn't get done and why

- [ ] "real" testing via mirroring traffic to new service
    - still cleaning up results of local/integration tests, not ready for prod traffic just yet
- [ ] Continue to look at growth metrics and see what other properties we can add to better understand our growth @dmarticus 
    - didn't prioritize
- [ ] Feature mgmt progress, support work took over majority of time

---

## Sprint

### High priority

- [ ] testing/productionizing new `/flags` service @dmarticus 
    - [ ] integrate error spans with new error tracking service
    - [ ] implement holdout group targeting
    - [ ] add more metrics for production testing 
    - [ ] "real" testing via mirroring traffic to new service
- [ ] Feature mgmt crud working end to end, get feedback @havenbarnes 

### Side Quests

- [ ] enforce singleton requirements for the python SDK @dmarticus 
- [ ] Continue to make sentry more actionable and useful @havenbarnes 
- [ ] Continue to look at growth metrics and see what other properties we can add to better understand our growth @dmarticus 
- [ ] Fix team_id label on feature flags grafana dashboard @havenbarnes 
- [ ] (Look into) adding `/decide` trx id to $feature_flag_called event @havenbarnes 


#### Things to chew on

- [ ] Can we know when people are starting to make new features?

## OKRs

1. Ship new feature flags in production and accomplish the 10x scale + 2x speed boost performance targets (ideally decreases infra spend by N% as well, will try to figure out a more specific target going forward)
2. Maintain current /decide service and make optimizations.

**Comment by benjackwhite** [comment]
# Team Christmas Don't Panic (CDP)
- Support: @benjackwhite 

Rough time off: ben 50%, Marius 100%, Marcus 50%, Olly mixed

## Retro

NOTE: We swapped a few things around mid sprint to support some big customer imports

* 🟡 @oliverb123 - making strong progress on batch import work so that we can handle these kind of requests
* 🟢 @benjackwhite gradually roll out RemoteConfigs to everyone (providing the code for site-functions)
  * Only final bits needed just want to safely gradually roll out
* 🟡 @benjackwhite Setup internal event source destinations
* 🟢 @mariusandra finish Mappings work with focus on TikTok Ads
  * Mappings are out and working just blocked by RemoteConfig work
* 🟢 @MarconLP build out and test TikTok Ads / Snapchat client side integration
  * Still waiting on google ads to verify for standard access 
 
## OKRs
* 🟢  Widespread usage of the new CDP 
* 🔴 Messaging V1 
* 🟡 Ingestion is rock solid

### High priority

* Keep things stable and running!
* Finish the internal event source destinations @benjackwhite 
* Rollout RemoteConfig when it feels very safe (or just get more metrics to rollout post-break) @benjackwhite 
* Tiktok / snapchat client side destinations - running on an actual website of some kind! @MarconLP (could chat with marketing team to see if there is a good thing we could test this with)
* Batch imports @oliverb123 - getting at least one of the target customers imported via this.
* Be ready to onboard 2 new people!

### Stretch goal
* Server side mappings

**Comment by daibhin** [comment]
# Team Error Tracking

@daibhin off (most days)


## Retro

🟢 Launch closed beta @daibhin 
30 teams sent at least 1 exception event in the last 24 hours
Updated docs with info on exception sampling & custom grouping
Billing repo still can't be upgraded so we're looking into it. Need to find some overlap with Zach

🟢 Cymbal improvements @oliverb123
Throughput not a problem since adding Kafka write batching. Also solves health check issues
Memory leak solved (cache invalidation is hard when you don't invalidate the cache)
Alerting & monitoring is still something we need to create a dashboard for

🟢 Alerting @daibhin
Decided on approach with Ben & splitting up the work. Should have issue action (created, reopened) alerts by EOW

🟢 Improved UI for issues page should be ready by EOW @daibhin

## OKR

✅ Human readable stack traces for Javascript
✅ Ensure scalability of query time issue grouping
🟢 Alerting via CDP
🟡 Launch

### High priority

- [ ] Decide on Q1 goals
- [ ] Deal with any beta issues

**Comment by pauldambra** [comment]
# Team (R)One (E)Door (P)Closes (L)and (A)nother (Y)Opens

xmas sprint

**Support hero:** @pauldambra 
Manoel is off, Paul is off, Yiannis is flying solo/yolo

<details>
<summary>Retro</summary>

## what we shipped


* 🍾 big golden bottle of champagne in london to celebrate the final platform being released
	* we completely forgot to drink champagne 🫠 how?! 
* size of mobile replay @marandaneto @ioannisj 
	* did diff detection
	* ready to switch to webp in january
* ✅ manual API for starting and stopping all 4 platforms @marandaneto @ioannisj  
    * pending but should be done (particularly since picking up some surveys work)
    * needs `reset`, `resetSessionId`, `startRecording`, `stopRecording`
    * needs full snapshot on resume in mobile
* 🔄 Capture network headers and body (assuming new instrumentation is in place by then) https://github.com/PostHog/posthog-ios/issues/159 @ioannisj 
* ✅ long sessions on mobile - https://github.com/PostHog/posthog-ios/issues/227.  @ioannisj 
   * should be in by Friday 
* ✅ handover from @Phanatic 

## Replay

* replay 4000 UI 
   * ✅ improve overview tab using sessions data 
* 🔄 a brief diversion before querying network data - wrap websockets as well as fetch @pauldambra
    * wrapping websockets turns out to be a pain 	

### Low priority / side quests

</details>

<details>
<summary> OKRs 🟡 </summary>

1. OKR, status (red/yellow/green) and action points if yellow/red

🟢 📱Goal 1: People think of PostHog as a mobile solution
🟡 📹 Goal 2: Replay 4000
    - this is now so excellent it's Replay 4000
    - https://github.com/PostHog/posthog/issues/26098
🟡 🎤 Goal3: Ship the things users keep asking for
 - [ ] queryable network data
 - [x] start replay on trigger e.g. event
 </details>

### High priority

* brandy butter
* brandy cream

## Mobile

* 🔄 Capture network headers and body (assuming new instrumentation is in place by then) https://github.com/PostHog/posthog-ios/issues/159 @ioannisj 

### Low priority / side quests

* (can we|do we)  measure watch time and buffer time? @pauldambra 
* go through support and find people who wanted block list and event trigger @pauldambra 
* 🔄 get people using iOS autocapture @ioannisj and @marandaneto

**Comment by MichaelKutsch-ph** [comment]
# Team Infrastructure

Christmas, Christmas, Christmas

<details>
<summary> Retro </summary>
## Retro

- 🔴  100% do the reservation changes ASAP - @danielxnj 
  - we want to wait until the clickhouse migration is done so we see the actual feasible spend
  - discussions about reseller / PPA with AWS has a huge impact on this decision, we could do some wrong decisions here if this is not finalized first
- 🟢  Optimize the LB/envoy deployments to remove all cross AZ traffic (cost optimization) @MichaelKutsch-ph 
	- has positive effects on the bill, but not as impactful as expected -> loads of work on every tiny part, therefore better focus on pricing agreements
- 🟡 ArgoCD - move lowest priority posthog services over (cdp, temporal-workers, cymbal) @bciaraldi 
  - Cymbal is deployed 👍 
  - we had some other issues that were more important to resolve, therefore prioritized down
  - A bunch of stuff missing on our internal cluster to make this stuff super robust
- updating VM to latest to resolve alerting issues
	- challenging and time consuming task - but we prioritized this as we have issues with alerting
	- we have to talk to VM to figure out some dependencies
- initial discussions with nOps, Pump and DoIT 
</details>

<details>
<summary> OKR  </summary>
## OKR

1. 🏃 Continuous Delivery
2. 💰 Cost visualization
3. 🧪 Continuous Integration
</details>


### High priority

- install nOps agent to detect x-az traffic issues @danielxnj 
- dive into details and decide about AWS Pricing @MichaelKutsch-ph 
- Lock in WS @timgl 
- Continue w/ internal cluster into the same state as our prod clusters (monitoring stack, SSO access etc.) @danielxnj 
- ArgoCD @bciaraldi 
  - some prep stuff to do for the cluster
  - start moving apps to argo in parallel @MichaelKutsch-ph 	
	  - Get PostHog  web (and workers) onto ArgoCD 
- when Frank is back: @benjackwhite 
	- plan move to Warpstream for capture/events
- exhaustion of IPs on prod: looking into it @bciaraldi 

### Low priority / side quests

**Comment by marandaneto** [comment]
## Team Farewell (Surveys)

@Phanatic offboarding December 23rd
@marandaneto PTO the whole sprint
@ioannisj a few days PTO
Lucas joining the team

### Retro

* :question: Fixing bugs reported by Annika and a few customers that we're getting more survey responses than allowed when using Adaptive Sampling.
  - @Phanatic I don't see any PRs related to that, did you get to it?
* :question: Fixing bugs reported by customers of NPS recurring surveys who are confused about when the survey is shown or hidden.
  - @Phanatic I don't see any PRs related to that, did you get to it?
* :question: Handing off no-code experiments to the Experiments team.
  - @Phanatic or @PostHog/team-experiments did this happen?
* ✅ Handing off Surveys to @marandaneto and @ioannisj 
* ✅ Customer support
  - Phani is OOO, Manoel and Dylan are on it
* :no_entry_sign: Q1 Planning
  - Early January

### Sprint goals

* Customer support @dmarticus and @ioannisj
* Onboarding Lucas @dmarticus

**Comment by robbie-c** [comment]
# Team web analytics

**Support hero:** @rafaeelaudibert 

@rafaeelaudibert: Working 26th & 27th Dec
@robbie-c: Working 2nd & 3rd Dec

<details>
<summary>Retro</summary>

Robbie
🟡 Finish cookieless prototype

Rafael
🟢 Conversion goal
🟢 Add more options for previous period comparison (use product analytics UI)
🟢 Comparison to the previous period on all queries
🟢 Get comfortable doing web analytics support
🟢 Some nextjs improvements
</details>

<details>
<summary>OKR</summary>

🟠 Continue to improve performance
🟢 Build a small amount of more advanced marketing features, to measure interest
🟡 Cookieless tracking prototype
🔴 Improve the NextJS experience
🔴 More page-level insights
🇬🇦 Improve the documentation, and link to it more from the app
</details>

### High priority

Rafael
* Christmas support tickets
* Core web vitals product research

Robbie
* Finish cookieless prototype
* New year support tickets

**Comment by fuziontech** [comment]
# Team ClickHouse

## OKR Q2 2024

### Objective

**Goals for Q4:**

- Improve elasticity and flexibility of our data store by putting all our data in Iceberg
    - [Work with Altinity](https://docs.google.com/document/d/12icx71-_kV0aICP3igtOc0di_yrey_W82HzbadtNPBY/edit?usp=sharing) effectively to ship read path for Iceberg on ClickHouse - Brett Hoerner
    - Setup infrastructure to ship all of our data to Iceberg on S3 - James Greenhill
    - Shipping Query logs to Iceberg - James Greenhill
- Continue improving CH operational expertise
    - Upgrade to a later version of Clickhouse - James Greenhill
    - Capacity planning - James Greenhill
    - Automation - Daniel Escribano
    - Put some of the basic mitigation operations in runbooks - Daniel Escribano
- Schema management
    - Tool for schema migration (coordinator schemas) - Daniel Escribano
    - Tool for long running mutations - Daniel Escribano
- Continued investment in performance
    - Tooling for other teams to understand which queries are slow and why - Ted Kaemming
    - Investigate variability of queries - Ted Kaemming
    - Per-team limits on queries/query complexity (needs product work) - Ted Kaemming

## Board
https://github.com/orgs/PostHog/projects/85/views/2

## Retro

### 🔥 Incidents 🔥 
**More than normal this cycle**
- [#inc-2024-12-11-ingestion-delays](https://posthog.slack.com/archives/C084NJBPLRY)
- [#inc-2024-12-11-clickhouse-node-down-us](https://posthog.slack.com/archives/C084Z4RLS7K)
- [#inc-2024-12-10-bad-replay-migration](https://posthog.slack.com/archives/C08541Y824Q)
- [#inc-2024-12-06-us-clickhouse-slowness](https://posthog.slack.com/archives/C084EKPDQ1X)

### Last Sprint's Goals
~~~ Cost investigation ~~~
- Review increasing cost on all AWS @fuziontech 🟡 
  - Cleared up a lot of EBS, but still have two clusters (will be one by Friday)
~~~ Validate and move over to new cluster quickly ~~~
- Validate cluster data parity between old and new clusters @fuziontech 🔴 
  - Clusters diverged, then an outage prevented us from reconciling. 🔥 
- Fix deletes @fuziontech 🔴
- Benchmark to new 8 shard lower cost cluster w/ coordinator nodes @fuziontech 🟢 
  - This one was actually done and early indications look good.
~~~ Mat Col optimizations ~~~
- Investigate how to speed up index dropping for Mat Cols @tkaemming  🟢
  - Increased thread pool available to mutations 😎 
- Yeet unused mat cols @tkaemming  🟢
~~~ Below the fold ~~~
- Investigate what needs to be done for queue metering / QoS @fuziontech  🟢
  - https://posthog.slack.com/docs/TSS5W8YQZ/F0855S0TBPE
- Get query logs into iceberg. @fuziontech 🔴
- (low priority) Expose Iceberg table for testing @fuziontech 🔴
	- Current version this will be quite slow
	- New 24.8 version will be quite fast

## This Sprint's Goals 

### Holiday plans
@fuziontech - 
@tkaemming - 
@Daesgar -
@orian -

### TL;DR

Not much - KTLO ideally

**Comment by EDsCODE** [comment]
# Team Data Storehouse
### OKR Q4 2024

### Objective

**Query 3000**

* **Key Results:**
    * Autocomplete
	* Increase general BI experience/product https://github.com/PostHog/meta/issues/157
	* Declutter the data warehouse UI and make the features intuitive to find

**Data Modeling MVP**

* **Key Results:**
	* Infrastructure decided and implemented
	* Integrating external data with feature flags
	* External data everywhere in insights/persons/cohorts
	* Get billing team to use modeling in posthog for their invoices_with_annual table
 
### Retro
Tom off for most of sprint

- [ ] Draft queries + reorg sidebar (https://github.com/PostHog/posthog/issues/26193) @EDsCODE 
- [x] Materialization Tab (https://github.com/PostHog/posthog/issues/26193) @EDsCODE 
	- refactored most of new editor logic
- [ ] Permanent search bar
- [ ] Release data import to everyone @Gilbert09 
	- Side quest to work through bigquery integration for customer. Import being testing tomorrow

### Priority

Both off most of first week

- [ ] Draft queries + reorg sidebar (Epic: Editor & Modeling 3000 #26193) @EDsCODE
- [ ] Data import deployment and testing @Gilbert09

**Comment by aspicer** [comment]
# Team Product Analytics
## Q4 2024 objectives
**Legacy 0x** - Thomas owning
* All insights are query-based
* rm -rf posthog/queries/
* Insight query is stored in the URL for instant sharing of unsaved insights

⠀**First-class environments** - Michael owning
* Environments are rolled out to everyone

⠀**Threshold-based alerts** - Anirudh owning
* All users get to set up alerts on PostHog metrics
* Alerts are launched with Marketing, and positioned into paid plans
* We're using alerts ourselves for key trends and funnels

⠀**Flexible funnels** - Sandy owning
* UDF funnels rolled out to everyone, unlocking arbitrary complexity
* Allowing parts of funnels to be unordered. No more separation between regular, unordered, and strict funnels
* Optional funnel steps

⠀**10x onboarding and ease of discovering features** - Anirudh and Sandy owning
* We run an exercise in implementing PostHog in a hobby project. What stands out as painful or unclear?

⠀**User delight 2x** - support rotation owning
* Support is 1 hero + 1 sidekick
* Every week support folks ship those small requested features (see [~[list of requests so far](https://docs.google.com/spreadsheets/d/1-lXjfzo6313Ai2vMf6smmoA0zghitBQTRD4lPgZ-XqA/edit?gid=0#gid=0)~](~[https://docs.google.com/spreadsheets/d/1-lXjfzo6313Ai2vMf6smmoA0zghitBQTRD4lPgZ-XqA/edit?gid=0\#gid=0](https://docs.google.com/spreadsheets/d/1-lXjfzo6313Ai2vMf6smmoA0zghitBQTRD4lPgZ-XqA/edit?gid=0#gid=0)~))
* Goal: 90% of tickets fulfill the SLA

⠀**B2B analytics research** - Anna owning
* Understand the market and user needs better in order to improve our current Group analytics product next quarter

🟢🟡🔴
# Retro
**Support hero:** **~[@aspicer](https://github.com/aspicer)~** **~[@thmsobrmlr](https://github.com/thmsobrmlr)~**
**SDK Support hero**: **~[@anirudhpillai](https://github.com/anirudhpillai)~** **~[@aspicer](https://github.com/aspicer)~**
* Alerts [Anirudh]
  * Scaling with the rollout
    * Improving execution system for running alerts
    * Bugfixes and deploy monitoring
* Carryover: Cached Dashboards, further rollout [Anirudh]
  * Thinking about API concurrency and scaling
* 🟢 Peter Onboarding [Sandy]
* 🟡 Carryover: Cohort migration production monitoring and bug fixing [Sandy]
* 🟢 Preparing for Planning [Anna]
  * Collect build better notes and user interviews for Q1 Planning
* 🔴 Migrate api endpoints to use hogql queries [Thomas]

# Sprint Plan

**Support hero:** @anirudh @aspicer
* Monitor and fix issues with hogql cohort calculation [Sandy]
  * Stretch goal: roll it out as primary cohort calculation logic
* Dashboard creation / edit experience improvements [Peter]
* Rollover: Migrate api endpoints to use hogql queries [Thomas]

**Comment by zlwaterfield** [comment]
# Team Growth

## Retro
<details>  
<summary>Retro items</summary>

- @zlwaterfield 
	- Support hero week 1
	- [x] Misc fixes from RBAC middleware changes
	- [ ] RBAC UI behind a feature flag
	- [x] Finish the 2FA improvements
	- [ ] Block usage reporting when on a trial (request from sales)
	- [ ] Finish billing postgres db improvements - pgbouncer and read replica -> infra is working on this
- @surbhi-posthog
	 - Support her week 2
	 - [ ] custom sql analysis on ICP data
	 - [x] load icp properties into Posthog?
	 - [x] Add events for Team's features in the backend
- @patricio-posthog 
	- [x] Keep monitoring the new forecasting, check that everything runs smoothly
	- [ ] Update `/billing` endpoint to include the new forecasting for customers
	- [ ]  API query pricing
	- [ ] Start working on the new pricing for mobile [RFC](https://github.com/PostHog/product-internal/pull/669)
</details>  

## Q4 Goals
✅=finished 🟡=in progress 🔴=won't finish ⚪=not started
1. 🟡 Get to a 4/10 in Enterprise platform support (Zach)
	- 🟡 Release Enterprise addon and iterate with sales to find a pricing model that makes sense
	- 🟡 Build RBAC
2. 🟡 Make revenue & forecasting more reliable (Patricio)
	- ✅ Rejig how we do sync_invoices so we don’t see big drops - respond to webhooks instead of large syncs
	- 🔴 Evaluate temporal vs celery for usage reports and billing tasks so we can adhere to rate limits better
	- 🔴 Side hustle, if time permits: Rejig how anonymous / identified events are billed, as the current implementation is confusing.
3. 🟡 Activation: reverse trials, analytics onboarding flow, understand CC activation (Raquel)
	- ✅ Dig into the data, watch replays to figure out why CC activation rate has been falling
	- 🔴 Add reverse trial to onboarding
	- ✅ Add reverse trial to billing page for teams

## This sprint

Time off: lots of time off with xmas and such

- @zlwaterfield 
 	- [ ] RBAC focused 
 	- [ ] Customer.io emails in PostHog - RFC open
 	- [ ] 2FA emails
- @surbhi-posthog 
 	- [x]  ICP data migration from salesforce to Posthog
	- [X]  New ICP Growth Dashboard. This is just the beginning, and I need to keep working on validation data quality and definitions
 	- [X] Helped with shipping 2fa PR
 	- [ ]  Product analytics hypothesis to help low ICP users. I have a PR out, and looking for feedback
 	- [ ]  Update docs about data migration learnings
 	- [ ]  Build final 2 events for team features.
- @patricio-posthog 
	- [ ] PR for forecasting in the frontend is ready, once merged I'll monitor it https://github.com/PostHog/billing/pull/1046
	- [ ] Create new pricing on Stripe for Mobile Pricing
	- [ ] Update trust scores when customer updates their limits
	- [ ] Tool in Billing admin to assign credits to list of emails (ref: ElevenLabs credit program)

**Comment by fuziontech** [comment]
# Team ClickHouse

## OKR Q2 2024

### Objective

**Goals for Q4:**

- Improve elasticity and flexibility of our data store by putting all our data in Iceberg
    - [Work with Altinity](https://docs.google.com/document/d/12icx71-_kV0aICP3igtOc0di_yrey_W82HzbadtNPBY/edit?usp=sharing) effectively to ship read path for Iceberg on ClickHouse - Brett Hoerner
    - Setup infrastructure to ship all of our data to Iceberg on S3 - James Greenhill
    - Shipping Query logs to Iceberg - James Greenhill
- Continue improving CH operational expertise
    - Upgrade to a later version of Clickhouse - James Greenhill
    - Capacity planning - James Greenhill
    - Automation - Daniel Escribano
    - Put some of the basic mitigation operations in runbooks - Daniel Escribano
- Schema management
    - Tool for schema migration (coordinator schemas) - Daniel Escribano
    - Tool for long running mutations - Daniel Escribano
- Continued investment in performance
    - Tooling for other teams to understand which queries are slow and why - Ted Kaemming
    - Investigate variability of queries - Ted Kaemming
    - Per-team limits on queries/query complexity (needs product work) - Ted Kaemming

## Board
https://github.com/orgs/PostHog/projects/85/views/2

## Retro

### 🔥 Incidents 🔥 
**More than normal this cycle**
- [#inc-2024-12-11-ingestion-delays](https://posthog.slack.com/archives/C084NJBPLRY)
- [#inc-2024-12-11-clickhouse-node-down-us](https://posthog.slack.com/archives/C084Z4RLS7K)
- [#inc-2024-12-10-bad-replay-migration](https://posthog.slack.com/archives/C08541Y824Q)
- [#inc-2024-12-06-us-clickhouse-slowness](https://posthog.slack.com/archives/C084EKPDQ1X)

### Last Sprint's Goals

~~~ Cost investigation ~~~
- Review increasing cost on all AWS @fuziontech 🟡 
  - Cleared up a lot of EBS, but still have two clusters (will be one by Friday)
~~~ Validate and move over to new cluster quickly ~~~
- Validate cluster data parity between old and new clusters @fuziontech 🔴 
  - Clusters diverged, then an outage prevented us from reconciling. 🔥 
- Fix deletes @fuziontech 🔴
- Benchmark to new 8 shard lower cost cluster w/ coordinator nodes @fuziontech 🟡 
  - This one was actually done and early indications look good.
~~~ Mat Col optimizations ~~~
- Investigate how to speed up index dropping for Mat Cols @tkaemming  🟢
  - Increased thread pool available to mutations 😎 
- Yeet unused mat cols @tkaemming  🟢
~~~ Below the fold ~~~
- Investigate what needs to be done for queue metering / QoS @fuziontech  🟢
  - https://posthog.slack.com/docs/TSS5W8YQZ/F0855S0TBPE
- Get query logs into iceberg. @fuziontech 🔴
- (low priority) Expose Iceberg table for testing @fuziontech 🔴
	- Current version this will be quite slow
	- New 24.8 version will be quite fast

## This Sprint's Goals 

### Holiday plans
@fuziontech - Vancouver 24->26
@tkaemming - OOO 23 -> 27 travel 30
@Daesgar - <fill>
@orian - OOO 23 -> 27

### TL;DR

Not much - KTLO ideally

- Benchmark new cluster @fuziontech 
- Shut down new cluster @fuziontech 
- Get into a place where we can quickly restart new cluster @fuziontech 
- chproxy into dev stack @orian 
- Look into weekend event delete job @orian 
- Look into iceberg for query log @orian
- Squash job @tkaemming
